string plugin_name
int32 n_regs
---
string[] regs_of_space_id